### PR TITLE
Add icon support

### PR DESCRIPTION
**Pull request by jason-fox**
- Update Plugin Code
- Update Docs
- Add Sample Footer with icons
- Add icons to `<note>` elements

Screenshot 2021-07-09 at 11 44 59

The actual sample footer is found in #10 and is more elaborate than this screenshot, and like the header contains dummy links and texts.

### CODE REVIEW

**Review by infotexture — CHANGES_REQUESTED** [review]
When I check out this branch and build the DITA-OT docs as sample output, the build fails with XSLT errors:

```console
[xslt] Error evaluating ($BOOTSTRAP_ICONS_CDN) at procedure BOOTSTRAP_ICONS_CDN on line 15 of html5-bootstrap.xsl:
[xslt]   XTDE0700: A value must be supplied for parameter $BOOTSTRAP_ICONS_CDN because there is no
[xslt]   default value for the required type
```

This can probably be avoided by specifying the missing value, but this shouldn't be **required**.

Not everyone will want or need these icons, so it should still be possible to build output without them.

File: sample/icons.dita [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+<!DOCTYPE topic PUBLIC "-//OASIS//DTD DITA Topic//EN" "topic.dtd">
+   <!-- Within the sample documentation, where necessary, the texts describing the
+   usage of each component have been copied directly from the official Bootstrap
+   5.0 documentation (found at https://getbootstrap.com/docs/5.0), however DITA
+   markup is used throughout the examples describing how to implement these
+   components correctly using outputclass. -->
+<topic id="icons">
+  <title>Icons</title>
+  <shortdesc>
+    DITA Bootstrap includes the Bootstrap Icons set by default. Other Icon libraries such as
```
Comment (infotexture) [inline sample/icons.dita:11]:
```suggestion
    DITA Bootstrap includes Bootstrap Icons by default. Other icon libraries
```

Comment (jason-fox) [reply sample/icons.dita]: Fixed cfe1e62

**Comment by jason-fox** [comment]
> When I check out this branch and build the DITA-OT docs as sample output, the build fails with XSLT errors:

It wasn't erroring for me, so maybe a Windows issue. The default had been left as `'yes'` rather than `'no'` and I've moved the CDN PATH into the if clause. Hopefully fixed cfe1e62

File: .github/dita-ot/header.xml [hunk 2]
```diff
@@ -1,7 +1,14 @@
...
... 412.66016 50.58203C 416.37296 46.70021 425.91992 44.759766 425.91992 44.759766zM282.10547 124.49805C 274.74503 124.26863 266.17072 131.36765 261.70703 136.76172C 250.14606 150.73245 247.87634 154.7875 241.73047 164.32617C 235.31197 174.288 229.14072 184.47758 224.13867 195.2207C 215.13596 214.55627 208.04654 234.79895 201.96484 255.24219C 198.37534 267.30804 194.87244 280.5286 191.56055 295.93555C 184.73207 327.70187 181.82349 342.04816 181.26562 349.71484C 181.0422 352.78516 182.17294 351.39395 183.20312 350.3828C 197.70508 336.14902 209.89832 320.03864 216.93164 311.20312C 224.44554 301.76385 230.80168 293.37238 237.44531 284.24023C 264.92267 246.42532 270.74084 241.54958 286.64844 212.24023C 304.34872 179.62787 308.04166 139.1942 286.9414 125.93945C 285.43762 124.9948 283.80402 124.55099 282.10547 124.49805zM444.79297 319.0254L444.79297 408.55078C 444.79297 428.80496 428.48865 445.11133 408.23438 445.11133L408.23438 445.11133L354.72656 445.11133L354.72656 391.9336C 356.6592 390.95056 358.79715 389.59668 360.80273 387.73047C 360.93594 387.65097 361.06686 387.5826 361.20117 387.49805L361.20117 387.49805L444.79297 319.0254z" stroke="none" fill="currentColor" fill-rule="nonzero" />
```
Comment (infotexture) [inline .github/dita-ot/header.xml:7]: I like the idea of including a logo here in the header, but we shouldn't use the DITA-OT logo, as this is a separate project.

I'll push a commit to replace this with the Bootstrap “B” as a placeholder for now.

**Review by infotexture — APPROVED** [review]

**Comment by infotexture** [comment]
P.S. I'd been meaning to add the DITA-OT logo to the header on the project website at dita-ot.org. 

Tested in my local fork and looks good, so I'll tweak a bit more and submit a PR to include it there.

**Pull request merged**